### DISCUSSION TITLE
Bare metal install

### DISCUSSION THREAD

**Discussion by svenvg93** [alexjustesen/speedtest-tracker#906] [Q&A] [answered]
I love this tools, thank you for making and maintaining it.
Was wondering is there is way to do a bare metal install instead of using docker. I would love to install it on the left over Pi for a friend, as it would be the only tool installed bare metal would be nice :)

**Comment by alexjustesen** [comment, marked as answer]
At this point (mentioned [here](https://github.com/alexjustesen/speedtest-tracker/issues/20)) I'm not planning on supporting bare metal installs but...

If you have a working knowledge of Laravel you could absolutely reverse engineer a setup.
